db/migrate/20220916091250_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string   :name
+      t.string   :email
+      t.string   :password
+      t.string   :address
+      t.string   :phone
+      t.datetime :date_of_birth
+      t.string   :gender
+      t.string   :type
+
+      t.timestamps
```

Comment:
indexes ?

Reply:
i will put indeces at the end of project

Reply:
resolved